rotkehlchen/tasks/utils.py

```diff
@@ -1,24 +1,22 @@
 from typing import TYPE_CHECKING
-from rotkehlchen.constants.timing import DATA_UPDATES_REFRESH
-from rotkehlchen.db.updates import LAST_DATA_UPDATES_KEY
+
 from rotkehlchen.serialization.deserialize import deserialize_timestamp
 from rotkehlchen.utils.misc import ts_now
 
 if TYPE_CHECKING:
     from rotkehlchen.db.dbhandler import DBHandler
 
 
-def should_check_data_updates(database: 'DBHandler') -> bool:
+def should_run_periodic_task(database: 'DBHandler', key_name: str, refresh_period: int) -> bool:
```

Comment:
you can limit the type at `key_name`. We know all of them so you can use `Literal`